collections/Laud_Misc/MS_Laud_Misc_306.xml

```diff
@@ -83,35 +133,299 @@
+                           </msItem>
+                           <msItem>
+                              <locus>(fol. 157r–v)</locus>
+                              <rubric>Sermo unde supra</rubric>
+                              <incipit>Natalem domini hesterna die celebravimus natalem servi hodie celebramus</incipit>
+                           </msItem>
+                        </msItem>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 157v–158v)</locus>
+                        <msItem>
+                           <title type="desc">Two sermons for the feast of the Holy Innocents (28 December)</title>
+                           <msItem>
+                              <locus>(fols. 157v–158r)</locus>
+                              <rubric>Sermo de innocentibus</rubric>
+                              <incipit>Christo igitur secundum prophetię fidem nato in Bethleem Iudę temporibus Herodis</incipit>
+                           </msItem>
+                           <msItem>
+                              <locus>(fol. 158r–v)</locus>
+                              <rubric>Item sermo ut supra</rubric>
+                              <incipit>Quocienscumque fratres karissimi sanctorum martyria celebramus</incipit>
+                           </msItem>
+                        </msItem>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 159r–v)</locus>
+                        <title type="desc">Sermon for Epiphany (6 January)</title>
+                        <rubric>Sermo Leonis pape de epiphania domini</rubric>
+                        <incipit>Dies dilectissimi quo primum gentibus salvator mundi apparuit</incipit>
+                        <explicit defective="true">serit discordias. ascendit iras. incitat lin</explicit>
+                     </msItem>
+                  </msContents>
+                  <physDesc>
+                     <objectDesc form="codex">
+                        <supportDesc material="perg">
+                           <support>parchment</support>
+                           <collation>
+                              <secFol>ra uitali</secFol>
+                           </collation>
+                        </supportDesc>
+                        <layoutDesc>
+                           <layout columns="2" writtenLines="30">Ruled in plummet for two columns of 30 lines.</layout>
+                        </layoutDesc>
+                     </objectDesc>
+                     <handDesc>
+                        <handNote script="protogothic">Protogothic bookhand.</handNote>
+                     </handDesc>
+                     <decoDesc>
+                        <decoNote type="colInit">Initials in plain red.</decoNote>
+                     </decoDesc>
+                  </physDesc>
+               </msPart>
```

Comment:
Peter didn't give a date for this section (unless as "12th century"). It might be worth email to ask if he also thinks this section like the preceding is s. xii 4/4. Certainly looks late 12th or even early 13th to me. Anyway I think we should add `history/origin` here